ush/forecast_det.sh

```diff
@@ -6,7 +6,7 @@ UFS_det(){
   echo "SUB ${FUNCNAME[0]}: Run type determination for UFS"
 
   # Determine if the current cycle is a warm start (based on the availability of restarts)
-  if [[ -f "${COM_ATMOS_RESTART_PREV}/${model_start_date_current_cycle:0:8}.${model_start_date_current_cycle:8:2}0000.coupler.res" ]]; then
+  if [[ -f "${COM_MED_RESTART_PREV}/${model_start_date_current_cycle:0:8}.${model_start_date_current_cycle:8:2}0000.ufs.cpld.cpl.r.nc" ]]; then
```

Comment:
This file is not present when `APP=ATM`, so its not an indicator for warm start.
Besides, one can "warm" start the atm, ocean and ice without this file if set correctly in the ufs_configure (I think).

Reply:
if you set it the mediator to a cold start in the ufs_confugire, you should be able to warm start everything but the mediator.

Reply:
updated